src/components/Dashboard/MonthlyGoal/MonthlyGoal.tsx

```diff
@@ -72,26 +72,27 @@ const MonthlyGoal = ({
   const { classes } = useStyles();
   const locale = useLocale();
   const [showHealthIndicator, setShowHealthIndicator] = useState(false);
-  const [usingMachineCalculatedGoal, setUsingMachineCalculatedGoal] =
-    useState(false);
+  const [machineCalculatedGoal, setMachineCalculatedGoal] = useState<
+    number | null
+  >(null);
 
   const toolTipText = useMemo(() => {
-    const formattedGoal = currencyFormat(goal, currencyCode, locale);
-    return usingMachineCalculatedGoal
+    return machineCalculatedGoal
       ? t(
           'Your current goal of {{goal}} is machine-calculated based on the past year of NetSuite data. You can adjust this goal in the settings preferences.',
-          { goal: formattedGoal },
+          { goal: currencyFormat(machineCalculatedGoal, currencyCode, locale) },
         )
       : t(
           'Your current goal of {{goal}} is staff-entered, based on the value set in your settings preferences.',
-          { goal: formattedGoal },
+          { goal: currencyFormat(goal, currencyCode, locale) },
         );
-  }, [usingMachineCalculatedGoal, goal, currencyCode, locale]);
+  }, [machineCalculatedGoal, goal, currencyCode, locale]);
 
-  const receivedPercentage = received / goal;
-  const pledgedPercentage = pledged / goal;
-  const belowGoal = goal - pledged;
-  const belowGoalPercentage = belowGoal / goal;
+  const monthlyGoal = machineCalculatedGoal ?? goal;
```

Comment:
Don't we need to use `goal` if it is available and default to `machineCalculatedGoal` if it is not?

Reply:
Oh, that is confusing. I would have expected there to always be a machine calculated goal, but to use user defined goal as an override.

Reply:
Oh OK. I feel like `machineCalculatedGoal/setMachineCalculatedGoal` is a confusing variable name because that variable is `null` if there is a staff-entered goal.

Reply:
I have re-constructed the components in order to make the code less confusing. _I also thought it was confusing and wasn't happy with it._